pkg/api/apipanic.go

```diff
@@ -24,12 +26,17 @@ func (h *APIPanicHandler) ServeHTTP(wr http.ResponseWriter, req *http.Request) {
 	defer func() {
 		if r := recover(); r != nil {
 			fields := logrus.Fields{
-				"panic_message": r,
-				"url":           req.URL.String(),
-				"method":        req.Method,
-				"client":        req.RemoteAddr,
+				"url":    req.URL.String(),
+				"method": req.Method,
+				"client": req.RemoteAddr,
 			}
-			log.WithFields(fields).Warn("Cilium API handler panicked")
+
+			if err, ok := r.(error); ok && errors.Is(err, syscall.EPIPE) {
+				log.WithError(err).WithFields(fields).Debug("Failed to write API response")
+				return
+			}
+
+			log.WithFields(fields).WithField("panic_message", r).Warn("Cilium API handler panicked")
```

Comment:
```suggestion
			log.WithFields(fields).WithField("panic_message", r).Warn("Cilium API handler panicked in an unexpected way")
```

Reply:
Hmm, to me the term panic already conveys that it is an unexpected event (I haven't actually introduced/changed this log message, just moved the `panic_message` field after removing it from the common ones). Happy to change the message if you feel the other clearer though.

Reply:
Ah that's a good point, we can leave it as is.